roxie/ccd/ccdfile.cpp

```diff
@@ -37,6 +37,10 @@
 #ifdef __linux__
 #include <sys/mman.h>
 #endif
+#if defined (__linux__) || defined (__APPLE__)
+#include <sys/syscall.h>
+#include <ioprio.h>
```

Comment:
I agree with mark on this one - theoretically <> and "" use different search paths.